### DISCUSSION TITLE
Stuck getting started

### DISCUSSION THREAD

**Discussion by nirvman** [coccoinomane/crabada.py#115] [Q&A] [answered]
I changed the web3 node back to avax with https://api.avax.network/ext/bc/C/rpc  and i get error staying that I dont have any teams. I wanted to get the bot working on Avax before I moved over to Swimmer. I already added a team to the .env file, added my wallet address and private key as well.

Here is my error. Thanks in advance.

src.common.logger - INFO - No mines to reinforce for user 0x...
src.common.logger - INFO - No mines to close for user 0x9...
src.common.logger - INFO - No available teams to send mining for user 0x..
src.common.logger - DEBUG - Waiting for 120 seconds.

**Comment by coccoinomane** [comment, marked as answer]
Hi @nirvman , and welcome to the community!

Crabada.py is now Swimmer only.

If you want to test it out on Avalanche C Chain, you need to pull the commit before Swimmer compatibility was introduced.

Cheers,
Cocco
